### ISSUE TITLE
Admin Translations downloads error

### ISSUE THREAD

**Issue by mrsaicharan1** [fossasia/open-event-frontend#3335]
**Describe the bug**
Downloading admin translations gives the following error 
```
127.0.0.1 - - [07/Jun/2019 21:13:30] "OPTIONS /v1/admin/content/translations/all HTTP/1.1" 308 -
```

**To Reproduce**
1. Go to 'Admin Panel`
2. Click on 'content'
3. Click on download translations

**Expected behavior**
Should download zip file

**Additional details (please complete the following information):**
 - MacOS Mojave

**Additional context**
Working on it

**Comment by CosmicCoder96** [comment]
@kushthedude @uds5501 It's never ~lupus~  CORS.
When there is an internal error on serve, it fails to return anything, which throws a CORS exception, as 500 is not an acceptable status code.

**Comment by kushthedude** [comment]
Closed in #3336
